api/api.go

```diff
@@ -14,12 +14,29 @@ const (
 	apiTimeout = 5 * time.Second
 )
 
+// HttpGET is a utility function for making http get requests with Sia-Agent set
```

Comment:
I would even prefer just plain `Get` and `Post`, since the meaning is fairly clear. (Actually, it's less clear inside the package; `api.Get` is much more obvious than `Get`)